### PR TITLE
Netcoreapp2.0 targetted TestPlatform

### PR DESCRIPTION
**Pull request by smadala**
- Moved vstest.console to target netcoreapp2.0.
- Removed not required anymore System.*.dlls.

@codito @livarcocc @eerhardt @sbaid

### CODE REVIEW

File: src/redist/redist.csproj [hunk 1]
```diff
@@ -154,18 +154,6 @@
           Condition=" '$(DISABLE_CROSSGEN)' == '' "
           AfterTargets="PublishMSBuildExtensions">
     <ItemGroup>
-      <NETCore10Assemblies Include="$(PublishDir)/System.ComponentModel.Primitives.dll;
```
Comment (eerhardt) [inline src/redist/redist.csproj]:
```
     <!-- Move the "1.0" assemblies back -->
     <Move SourceFiles="@(NETCore10Assemblies->'$(PublishDir)/%(Filename)%(Extension).bak')"
           DestinationFiles="@(NETCore10Assemblies)" />
```

should be removed.

File: NuGet.Config [hunk 2]
```diff
@@ -10,5 +10,6 @@
     <add key="xunit" value="https://www.myget.org/F/xunit/api/v3/index.json" />
     <add key="api.nuget.org" value="https://api.nuget.org/v3/index.json" />
     <add key="AspNetCurrent" value="https://dotnet.myget.org/F/aspnet-feb2017-patch/api/v3/index.json" /> 
+    <add key="vstest" value="https://dotnet.myget.org/F/vstest/auth/967ad631-d24c-4c0e-bece-922de3064956/api/v3/index.json" />
```
Comment (eerhardt) [inline NuGet.Config]: Why does this URL have a GUID in it?

Comment (eerhardt) [inline NuGet.Config]: It appears this is a pre-authenticated feed URL.  What permissions does that access token give?  This token is now publicly available on the internet and anyone can use it.  Could they upload/delete packages on that feed with it?

Why not make the feed a public feed like all our other feeds?

Comment (codito) [inline NuGet.Config]: It's readonly. The feed is public, we should use `https://dotnet.myget.org/F/vstest/api/v3/index.json`.

File: src/redist/redist.csproj (hunk 1 above)
Comment (smadala) [inline src/redist/redist.csproj]: @eerhardt In latest TestPlatform.CLI nuget package, doesn't have any NetCore10Assemblies.

Comment (eerhardt) [inline src/redist/redist.csproj]: See https://github.com/dotnet/cli/pull/6053/files#diff-acadc45977478b307ad6d53f23f6e17cR187.  You missed cleaning up usages of this Item you are removing.

Comment (smadala) [inline src/redist/redist.csproj]: Oops, Thanks @eerhardt will remove in next commit.

Comment (smadala) [inline src/redist/redist.csproj]: Done.

Comment (eerhardt) [inline src/redist/redist.csproj]: Thanks!

**Comment by livarcocc** [comment]
@dotnet-bot Test RHEL7.2 x64 Release Build

**Pull request merged**